File: patterns/c-patterns/pattern97.c
// Pattern 97: ...

//         *
//       # * #
//     * # * # *
//   # * # * # * #
// * # * # * # * # *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;

    int x = n;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j < 2 * n; j++)
        {
            if (j >= x && j < n + i && j % 2 == 1)
            {
                printf("* ");
            }
            else if (j >= x && j < n + i && j % 2 == 0)
            {
                printf("# ");
            }
            else
            {
                printf("  "); // Adjust spacing for better alignment
            }
        }
        x--;
        printf("\n");
    }

    return 0;
}

File: patterns/python-patterns/pattern97.py
# Pattern 97: ...

#         *
#       # * #
#     * # * # *
#   # * # * # * #
# * # * # * # * # *

n = 5

x = n

for i in range(1, n + 1):
    for j in range(1, 2 * n):
        if j >= x and j < n + i and j % 2 == 1:
            print("* ", end="")
        elif j >= x and j < n + i and j % 2 == 0:
            print("# ", end="")
        else:
            print("  ", end="")  # Adjust spacing for better alignment
    x -= 1
    print()

